### DISCUSSION TITLE
Broken homepage

### DISCUSSION THREAD

**Discussion by FaimMedia** [phalcon/phalcon#236] [General]
Dear Phalcon Friends,

Is there somewhere I can create a pull request for the [docs homepage](https://docs.phalcon.io)?

The search field has been broken for weeks, switching between version numbers in the header almost always results in a 404 page. When the search field did work, it would show results of all versions, which is irrelevant when already on a certain version.

I love Phalcon, but using a broken docs website makes development unnecessary frustrating.

Greetings, Tim

**Comment by niden** [comment]
@FaimMedia The repository you are looking for is https://github.com/phalcon/docs-app. The search is handled by Algolia. I guess I messed things up when I tried to upgrade to their latest JS code. If you can fix that search that would be great. There are also documents on how to do the autocomplete (from Algolia docsearch).

**Reply by FaimMedia** [reply]
Thanks! I will have a look at it this week!
